packages/web-app-files/src/views/shares/SharedWithMe.vue

```diff
@@ -166,9 +182,25 @@ export default defineComponent({
         )
       }
 
+      if (unref(filterTerm).trim()) {
+        const usersSearchEngine = new Fuse(result, { ...defaultFuseOptions, keys: ['name'] })
+        result = usersSearchEngine.search(unref(filterTerm)).map((r) => r.item)
+      }
+
       return result
     })
 
+    watch(filteredItems, () => {
+      if (!unref(areResourcesLoading)) {
+        markInstance.value = new Mark('.oc-resource-details')
```

Comment:
Please try to not always instance a new mark instance if possible. I was able to do so in our createShortcutModal instance